server/aws-lsp-partiql/build-scripts/compile-inline-tree-sitter-parser-wasm.mts

```diff
@@ -0,0 +1,13 @@
+import { readFileSync, writeFileSync } from 'fs'
+
+// Read .wasm binary file, NOTICE and LICENSE file and add description of our modifications.
```

Comment:
Nitpick, but this comment isn't fully accurate, we're not reading the NOTICE and LICENSE files

Reply:
Thanks for pointing out. Update it.